If the _content passed to the box_side macro is too long then it fails
Patch by Simon Branford (University of Birmingham)
--- cmake/neci_print_summary.cmake.orig	2023-01-28 12:12:05.844153000 +0000
+++ cmake/neci_print_summary.cmake	2023-01-28 12:12:26.698259000 +0000
@@ -28,9 +28,7 @@
 feature_summary( WHAT ALL INCLUDE_QUIET_PACKAGES)
 
 macro( box_side _content )
-    string( LENGTH "${_content}" _len )
-    string( SUBSTRING "                                                  |" ${_len} -1 _box_tail )
-    message( STATUS "  | ${_content}${_box_tail}" )
+    message( STATUS "  | ${_content} |" )
 endmacro()
 
 string(TIMESTAMP _time "%Y-%m-%d %H:%M")
